canonical-web-and-design/jp.ubuntu.com
# Update webpack config

## Done

- Update webpack config it is inline with other projects
- Add codecov for JS files

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8012/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that there is a `static/js/dist` directory which contains all the built JS files
- Check that the following commands work as expected:
  - `dotrun exec yarn lint-js`
  - `dotrun exec yarn test-js`
  - `dotrun exec yarn build`
  - `dotrun exec yarn watch-js`
